packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuPanel.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { AccordionPanel, AccordionPanelProps } from '../Accordion';
+import { DropdownExpandableMenuItemContext } from './DropdownExpandableMenuItem';
+
+export interface DropdownExpandableMenuPanelProps extends AccordionPanelProps {
+  testId?: string;
```

Comment:
If `AccordionPanelProps` already has `testId`, it doesn't need to be on this list.